src/vm/linux/types.rs

```diff
@@ -1 +1,75 @@
 // SPDX-License-Identifier: Apache-2.0
+
+pub const NR_CPUID_CONFIGS: usize = 12;
+
+/// CPUID_CONFIG is designed to enumerate how the host VMM may configure the
+/// virtualization done by the Intel TDX module for a single CPUID leaf and
+/// sub-leaf.
```

Comment:
This is equivalent to `struct kvm_tdx_cpuid_config` in the kernel. We should make note of that.